src/data/markdown/translated-guides/en/03 Using k6 browser/01 Overview.md

```diff
@@ -31,12 +31,28 @@ The main use case for the browser module is to test performance on the browser l
 <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
 ```javascript
-import { chromium } from 'k6/experimental/browser';
+import { browser } from 'k6/experimental/browser';
 import { check } from 'k6';
 
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
 export default async function () {
-  const browser = chromium.launch({ headless: false });
-  const page = browser.newPage();
+  const context = browser.newContext();
+  const page = context.newPage();
```

Comment:
Should we use `browser.newPage()` directly?

Reply:
See 4b0f34526d5fb0d93b5fafbc97e421e294ce44a2, also for your other related comments :)